registry/storage/schema2manifesthandler_test.go

```diff
@@ -57,9 +57,10 @@ func TestVerifyManifestForeignLayer(t *testing.T) {
 			errMissingURL,
 		},
 		{
+			// regular layers may have foreign urls
 			layer,
 			[]string{"http://foo/bar"},
-			errUnexpectedURL,
+			nil,
```

Comment:
I noticed this is a behavioral change. I can't think of a good argument against it. But I think I'd be slightly more comfortable if a different PR that explicitly made this change and was reviewed on its own merits.